### PR TITLE
footer-section

### PR DESCRIPTION
**Pull request by Manish-Kr-Saw**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -37,5 +37,15 @@ <h2>What have we learned so far</h2>
                 <li>Curently, we are learning git</li>
             </ul>
         </main>
+        <footer>
+            <div>
+                <p>
+                    <a href="#">Facebook</a>
```
Comment (puranik3) [inline public/index.html]: According to requirements, Facebook link is last. Please change.

**Review by puranik3 — CHANGES_REQUESTED** [review]
Please check and do the updates.

File: public/index.html (hunk 1 above)
Comment (Manish-Kr-Saw) [reply public/index.html]: I made the changes.

**Review by puranik3 — APPROVED** [review]
Ship it!

**Pull request merged**